org.ekstep.questionset-1.0/renderer/plugin.js

```diff
@@ -104,8 +104,8 @@ org.ekstep.questionsetRenderer = IteratorPlugin.extend({ // eslint-disable-line
     var question = undefined;
     var savedQSState = this.getQuestionSetState();
 
-    EkstepRendererAPI.addEventListener('renderer:content:review', function() {
-      this.reInstateQuestionsOnReview("soft");
+    EkstepRendererAPI.addEventListener("renderer:content:review", function(e) {
```

Comment:
Change the event name to "renderer:plugin:reset" by passing data in event as true/false for soft/hard reset.